### PR TITLE
feat: feature in feed page links to website

### LINKED ISSUE #665: Add details about GTFS features to the feed pages
### Describe the problem

Once the new gtfs.org is launched, we should add information about 

- What each feature is 
- How we detect a feature 

### Proposed solution

Include 

- The individual feature sections associated with each feature in the feed pages (e.g https://staging.gtfs.org/getting_started/features/base_add-ons/#shapes for the "Shapes" feature) 

- Include the Overview page link in a ["More info" link](https://staging.gtfs.org/getting_started/features/overview/) underneath the Features header

- https://github.com/MobilityData/gtfs-validator/blob/master/docs/FEATURES.md for those curious about the spec details

[List of features and associated documentation links here](https://github.com/MobilityData/gtfs-validator/pull/1812/files)

### Alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Alessandro100**
closes #665 
**Summary:**

On the Feed page, when you click on a feature it will open up the documentation page on gtfs.org

**Expected behavior:** 

On the Feed page, when you click on a feature it should open the correct documentation page based on the feature

**Testing tips:**

Use Chaimaa's analytics tool (https://qa.mobilitydatabase.org/metrics/gtfs/features) to find the features you want to test and make sure the features when clicked go to the right spot

**Discussion**
Right now we are able to map the features to the correct link using the name of the feature `ex: Levels`. This is not great as the name of the feature can change, and in certain instances the names are different from the documentation and the validation report `ex: Bikes Allowance vs Bike Allowed`. A possible solution to this issue would be to assign each feature an `id`. This would help with displaying information about the feature on the UI, but would also help track feature data if the name of the feature ever changes. Thoughts? 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [ ] Add or update any needed documentation to the repo
- [X] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [X] Linked all relevant issues
- [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)
- [ ] 
Screenshot 2024-09-05 at 08 02 46

### CODE REVIEW

File: web-app/src/app/utils/consts.ts [hunk 1]
```diff
@@ -1,170 +1,160 @@
...
+    fileName: 'pathways.txt',
+    linkToInfo:
+      'https://gtfs.org/getting_started/features/pathways/#pathway-details',
+  },
+  Levels: {
+    component: 'Pathways',
+    fileName: 'levels.txt',
+    linkToInfo: 'https://gtfs.org/getting_started/features/pathways/#levels',
+  },
+  'Traversal Time': {
```
Comment (Alessandro100) [inline web-app/src/app/utils/consts.ts:107]: From the validation report the name is `Traversal Time` but in the documentation it's referred to as `In-station traversal time`

ref
https://github.com/MobilityData/gtfs-validator/pull/1812/files

Comment (emmambd) [reply web-app/src/app/utils/consts.ts:107]: See above

**Review by emmambd — APPROVED** [review]
Looks great!

File: web-app/src/app/utils/consts.ts [hunk 2]
```diff
@@ -1,170 +1,160 @@
...
+    fileName: 'trips.txt',
+    linkToInfo: 'https://gtfs.org/getting_started/features/accessibility',
+  },
+  'Route Colors': {
+    component: 'Accessibility',
+    fileName: 'routes.txt',
+    linkToInfo:
+      'https://gtfs.org/getting_started/features/base_add-ons/#route-colors',
+  },
+  'Bikes Allowance': {
```
[the remark this answers is not in the document]
Comment (jcpitre) [reply web-app/src/app/utils/consts.ts:32]: my bad. This PR was never merged: https://github.com/MobilityData/gtfs-validator/pull/1728

[the remark this answers is not in the document]
Comment (emmambd) [reply web-app/src/app/utils/consts.ts:32]: That wouldn't fix the actual way it's coded in the validator though, so there's still work to do

**Review by qcdyx — APPROVED** [review]
LGTM

**Comment by cka-y** [comment]
This LGTM however i'm worried that the UI will need modification each time we introduce a feature on the validator side. If there's a problem with the naming, my opinion is that it should be fixed on the validator side and not mapped on the UI.

**Comment by emmambd** [comment]
Fair point. We'll have to add the documentation link anyways on the validator side as part of the next release. I'd suggest keeping this PR for now and then refactoring this once the feature links are actually in the validator for 6.0.

**Pull request merged**